frontend/views/containers/chatroom/ChatMain.vue

```diff
@@ -984,8 +984,7 @@ export default ({
             if (addedOrDeleted === 'ADDED' && this.messages.length) {
               const isScrollable = this.$refs.conversation &&
                 this.$refs.conversation.scrollHeight !== this.$refs.conversation.clientHeight
-              const fromOurselves = this.isMsgSender(this.messages[this.messages.length - 1].from)
-              if (!fromOurselves && isScrollable) {
+              if (isScrollable) {
```

Comment:
The logic here that does not auto-scroll if the message sender is the user himself/herself caused of the issue so I just removed it. (I guess the author missed the rare case where a user uses two browsers)

Reply:
Yeah... odd.